torchao/quantization/quant_api.py

```diff
@@ -630,7 +643,8 @@ def int8_dynamic_activation_int4_weight(
 
 
 def int4_weight_only(
-    group_size=128, layout=TensorCoreTiledLayout(inner_k_tiles=8), use_hqq=False
+    group_size=128, layout=TensorCoreTiledLayout(inner_k_tiles=8), use_hqq=False,
+    zero_point_domain=None
 ):
     """
     Applies uint4 weight-only asymmetric per-group quantization to linear layers, using
```

Comment:
nit: please update docs for zero_point_domain before landing

Reply:
update in https://github.com/pytorch/ao/pull/1401/commits/51a4505800e43fd2093098180c401bbd7a38a62d